### DISCUSSION TITLE
Directions Button not working on mobile

### DISCUSSION THREAD

**Discussion by lionssharedigital** [code4recovery/12-step-meeting-list#969] [General]
### Contact Details

jesse@lionssharedigital.com

### Website With Issue

https://austinaa.org/l

### What happened?

On iphone, the button opens up to the Maps app and takes me to my location. This is happening across all our meetings.

The page I need help with: https://austinaa.org/locations/hill-country-bible-church/?v=map

### TSML Version

Latest (default)

### Wordpress Version

Latest (default)

### What browsers are you seeing the problem on?

Chrome, Safari

**Comment by tim-rohrer** [comment]
Hi @lionssharedigital,

We ask users to please open Discussions. If necessary, we’ll open a Bug Report for our tracking.

I’m pretty sure I’m seeing what you’re seeing as well. I’m just not sure where the issue resides. We’ll take a closer look.

**Reply by lionssharedigital** [reply]
Thanks. Sorry about that. Appreciate your time and effort.

**Reply by lionssharedigital** [reply]
Can I please get some help with this? The client feels it's pretty urgent.

**Comment by joshreisner** [comment]
Hi @lionssharedigital sorry, we I believe I have the problem fixed in a pull request, hopefully we'll be able to get it released soon.

If you want to try applying the fix now, do you have access to edit files? If so, edit `wp-content/plugins/12-step-meeting-list/assets/js/public.min.js` by pasting in the contents of this file: https://raw.githubusercontent.com/code4recovery/12-step-meeting-list/a03fe03a0c48e13a0708a1d18d9138133cc1d23a/assets/js/public.min.js

**Reply by tim-rohrer** [reply]
I will try to look at this tonight. Thanks to @joshreisner, a fix has been found and applied. 

You might suggest the user install the Meeting Guide app on their phone? That is likely a better way to get directions in either case.

**Reply by joshreisner** [reply]
Yeah let's get this thing approved and released. The client is Gary, the intergroup manager for Hill Country AA. He's right to be concerned if the directions button isn't working for mobile users.

**Reply by joshreisner** [reply]
Hi @lionssharedigital version `3.14.7` was just released and should hopefully fix the issue, let us know if you run into problems

**Reply by lionssharedigital** [reply]
Works Great!
